tests/phpunit/tests/http/includeOldRequestsClass.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * Tests that the old Requests class is included
+ * for plugins or themes that still use it.
+ *
+ * @group http
+ */
+class Tests_HTTP_IncludeOldRequestsClass extends WP_UnitTestCase {
+
+	/**
+	 * Tests that the old Requests class is included for plugins or themes that still use it.
+	 *
+	 * @ticket 57341
+	 *
+	 * @coversNothing
```

Comment:
We have only single function for this class and i don't think we have any other function for this class so IMO we have to move this at class level eg. https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/pluggable/signatures.php#L1-L8

WDYT @costdev ?

Reply:
This annotation can be either at the class or method level, so we don't have to change this. Of the 8 instances of `@coversNothing` in the test suite (excluding this one), 7 of them are at the method level, and only the example you linked above is at the class level. For consistency with the majority of the test suite, I'd prefer to keep this annotation where it is for now.